experimental/dds/sequence-deprecated/api-report/sequence-deprecated.api.md

```diff
@@ -14,6 +14,9 @@ import { IJSONRunSegment } from '@fluidframework/sequence/internal';
 import { IJSONSegment } from '@fluidframework/merge-tree/internal';
 import { ISegment } from '@fluidframework/merge-tree/internal';
 import { ISharedObject } from '@fluidframework/shared-object-base';
+import { ISharedObjectEvents } from '@fluidframework/shared-object-base';
+import { ISharedObjectKind } from '@fluidframework/shared-object-base/internal';
+import { ISharedObjectKind as ISharedObjectKind_2 } from '@fluidframework/shared-object-base';
```

Comment:
What's the reason for both imports, especially with one being internal?

Reply:
imports match in are identical in this package, so this is a new case of this problem for me. Fortunately, it just impacts API extractor reports (but maybe also docs? But we don't currently do docs for alpha/internal, so this should not matter). Thus I see no way to fix this, and no significant cost to leaving it, so I think I'll leave it, though it would be nice to understand better.

Reply:
Makes sense, it just seemed a bit odd to me but it seems fine to leave in then.